demisto_sdk/commands/download/downloader.py

```diff
@@ -344,7 +351,7 @@ def handle_file(self, string_to_write, member_name, scripts_id_name):
                 string_to_write, scripts_id_name
             )
 
-        if not self.list_files and re.search(PLAYBOOK_REGEX, member_name):
+        if not self.list_files and re.search(r"playbook-.*\.yml", member_name):
```

Comment:
why not use a regex?

Reply:
The change from @anara123 codes to the following line :
`if not self.list_files and re.search(PLAYBOOK_REGEX, member_name):`
caused the regression. 
Apparently PLAYBOOK_REGEX doesnt "catch" the playbooks type.

Reply:
can we fix it, instead? I guess It's used in more places

Reply:
I agree.
But not sure if its for the next SDK release.